frontend/src/model/Settings.ts

```diff
@@ -0,0 +1,20 @@
+type boardSize = 2 | 4 | 8 | 16 | 32 | 64;
+type tileAmount = 16 | 32 | 64 | 128;
+type turnLength = 5 | 10 | 15 | 30 | 60;
```

Comment:
There is a convention in typescript that types should be in PascalCase - please adjust.